module-system/module-implementations/sov-bank/src/hooks.rs

```diff
@@ -21,6 +21,12 @@ use crate::{Bank, Coins};
 // https://github.com/Sovereign-Labs/sovereign-sdk/issues/1234
 const GAS_TOKEN_ADDRESS: &'static str;
 
+#[config_constant]
+const GAS_TX_FIXED_COST: &[u64];
+
+#[config_constant]
+const GAS_TX_COST_PER_BYTE: &[u64];
```

Comment:
It looks like these constants are declared twice? (Once here, and once in transaction.rs). Is that intentional?

Reply:
Unintentional! Was removed a couple of seconds before the comment